### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by way0utwest** [dataplat/DataSaturdays#105] [Ideas]
I would propose that we need a default logo for events that people can use for swag, promotion, etc. While I do believe we ought to allow some events to upload or use their own, we ought to include a default.

My proposal is that we put this out the community and ask for a donation from an artist.

**Comment by spaghettidba** [comment]
Monica Rathbun suggested something like 99desings, so that we get proposals from multiple artists. 
Actually I already asked the designer next door to my office to draft something and I could cover that cost without a problem. The downside to that approach is that feedback would come late in the process compared to 99designs. I'll ping him today to see if he has some drafts to share

**Comment by spaghettidba** [comment]
I spoke with the designer next door and he hasn't done anything yet. (sigh...)
Regarding costs, he says that a logo with 3 possible proposals should cost around €500-€1000.
Sto start drawing something he also needs some ideas (similar logos we like, elements we want in the logo, the type (classic/modern), ...). If anyone wants to share some ideas that would be great. 
Another possibility is drafting a logo ourselves and he will fix colors, proportions and so on. This option costs less, but we have to do the creative process ourselves. As a backend guy, I'm allergic to design, but you may be more artistic than I am, so feel free to give it a try.

**Comment by rabryst** [comment]
Why complicate things?
![image](https://user-images.githubusercontent.com/3165117/106401834-8ed54680-63e3-11eb-9870-4b912baa1db2.png)

**Reply by sbitaxi** [reply]
My irrelevant $0.02 - I like this very much. It's clean, clear, will scale well, and it's purpose is obvious.

**Reply by amtwo** [reply]
I have some minor/detail changes I would make (and I'm happy to dive into those, if folks like the direction of this one)... but I generally like this logo & general direction.

**Comment by way0utwest** [comment]
I don't know that I think any particular logo matters, but we should choose something. I'm happy to ask for designs or just pick a few choices and vote on it. I would like to have us move forward in some manner, as well as having a way for individual events to override the logo.

**Comment by justinjbird** [comment]
FWIW, I'm a big fan of [Looka](https://looka.com/), have used it a few times for different things. You give it a few ideas and it uses AI (?) to offer you a range of style ideas. You can fiddle with a specific style so it's just right. You then just pay for the media, reasonably priced too. 

Perfect for design allergies :) Really nice interface...
<img width="1599" alt="Screenshot 2021-02-01 at 00 52 36" src="https://user-images.githubusercontent.com/42351666/106403954-daf6aa00-6428-11eb-8cf2-ccf4c9106c0e.png">

No affiliation yada yada...

**Comment by way0utwest** [comment]
I like the looka logo. Do you get rights to this, or do they retain some?

It would be good to get 2-3 ideas together for a vote and then close this.

**Reply by justinjbird** [reply]
Sorry @way0utwest only just noticed this. Found this page which gives a really good summary and answers your questions;

https://help.looka.com/en/articles/485739-what-s-included-in-each-of-the-logo-only-packages

I ran through the process for my initial contracting company and then did a second round for a venture that didn't materialise. It's a really nice platform, the interface is great for getting creative. I'm happy to walk folks through a workflow, but doesn't really warrant it as it's pretty intuitive...
